packages/integrations/mdx/src/rehype-images-to-component.ts

```diff
@@ -73,84 +73,105 @@ function getImageComponentAttributes(props: Properties): MdxJsxAttribute[] {
 
 export function rehypeImageToComponent() {
 	return function (tree: Root, file: VFile) {
-		if (!file.data.astro?.imagePaths?.length) return;
+		if (!file.data.astro?.localImagePaths?.length && !file.data.astro?.remoteImagePaths?.length)
+			return;
 		const importsStatements: MdxjsEsm[] = [];
 		const importedImages = new Map<string, string>();
 
 		visit(tree, 'element', (node, index, parent) => {
-			if (!file.data.astro?.imagePaths?.length || node.tagName !== 'img' || !node.properties.src)
-				return;
+			if (node.tagName !== 'img' || !node.properties.src) return;
```

Comment:
Why was this changed with fewer checks? Shouldn't we add a check for remote images?